das-cli/src/common/systemutils/sys_info.py

```
@@ -47,8 +47,8 @@ def get_disks_info(self):
                 partition_mntpoint = partition.mountpoint
 
                 generalDiskInfo = shutil.disk_usage(partition.mountpoint)
-                diskTotalMB = round(generalDiskInfo.total / 1024**2)
-                diskUsedMB = round(generalDiskInfo.used / 1024**2)
+                diskTotalMB = f'{(generalDiskInfo.total / 1024**3):.1f}'
```

Pylint (4.0.6)

[convention] Variable name "diskTotalMB" doesn't conform to '(([a-z][a-z0-9_]{0,})|(_[a-z0-9_]*))$' pattern

(C0103)